client/my-sites/checklist/wpcom-checklist/index.js

```diff
@@ -139,6 +188,29 @@ class WpcomChecklist extends PureComponent {
 						siteSlug={ siteSlug }
 						title={ translate( 'Give your site a name' ) }
 					/>
+					<TaskComponent
+						completed={ this.isComplete( 'email_verified' ) }
+						completedTitle={ translate( 'You validated your email address' ) }
+						description={ translate(
+							'To post and keep using WordPress.com you need to confirm your email address. ' +
+								'Please click the link in the email we sent at %(email)s.{{br /}}' +
```

Comment:
Should this be "...sent to..." instead of "...sent at..."?

Reply:
I think so too, but that's what the copy on the other banner is... I'll update it there....